docs/toc.md

```diff
@@ -13,10 +13,18 @@
             - [5. Get playing]({{urlRoot}}/projects/fps/get-started/get-playing)
             - [6. View your game world]({{urlRoot}}/projects/fps/get-started/view-game-world)
         - [Health packs tutorial]({{urlRoot}}/projects/fps/tutorial)
-    - Blank Starter Project
+    - Blank Project
         - [Overview]({{urlRoot}}/projects/blank/overview)
         - Setup
             - [Add a schema source]({{urlRoot}}/projects/blank/setup/schema)
+        - Tutorial
+            - [Overview]({{urlRoot}}/projects/blank/tutorial/overview)
+            - Blank Project 101
+                - [What is SpatialOS]({{urlRoot}}/projects/blank/tutorial/0/spatialos)
+                - [Key pieces]({{urlRoot}}/projects/blank/tutorial/0/key-pieces)
+                - [What it does]({{urlRoot}}/projects/blank/tutorial/0/what-it-does)
```

Comment:
Similar thing here, this is pretty vague. What is "it"? If I read these through as it stands I'd interpret it as: 

- What is SpatialOS?
- Key pieces (of SpatialOS)
- What it (SpatialOS) does
- Player lifecycle (in SpatialOS)
- Summary (of SpatialOS)